tests/rest_api/test_projects.py

```diff
@@ -0,0 +1,214 @@
+        response = get_method(username, f'projects/{project_id}')
+        assert response.status_code == HTTPStatus.FORBIDDEN
+
+    # [sandbox] Admin can see any project even he has no ownerships for this project (GET /projects/{id}).
+    def test_project_admin_accessibility(self, users, projects, find_users, is_project_staff, is_org_member, **kwargs):
+        admins = find_users(privilege='admin')
+
+        admin_project = self._find_by_owner(projects, admins, is_project_staff, is_org_member, False, False)
+        assert admin_project is not None
+        self._test_response_200(*admin_project[:2])
+
+    # [sandbox] Project owner or project assignee can see project (GET /projects/{id}).
+    def test_project_owner_accessibility(self, users, projects, find_users, is_project_staff):
+
+        for p in projects:
+            if p['owner'] is not None:
+                project_with_owner = p
+            if p['assignee'] is not None:
+                project_with_assignee = p
+
+        assert project_with_owner is not None
+        assert project_with_assignee is not None
+
+        self._test_response_200(project_with_owner['owner']['username'], project_with_owner['id'])
+        self._test_response_200(project_with_assignee['assignee']['username'], project_with_assignee['id'])
+
+    # [sandbox] Non-admin user cannot see project if this user is not project owner or project assignee
+    # [sandbox] (GET /projects/{id}).
+    def test_user_cannot_see_project(self, users, projects, find_users, is_project_staff, is_org_member):
+        non_admins = find_users(exclude_privilege='admin')
+
+        user_not_in_project = self._find_by_owner(projects, non_admins, is_project_staff, is_org_member, False, False)
+        self._test_response_403(*user_not_in_project[:2])
+
+    # [organization] (Organization context) Member of organization that has role supervisor or worker
+    # [organization] cannot see project if this member it’s not project owner
+    # or project assignee (GET /projects/{id}).
+    @pytest.mark.parametrize('role', ('supervisor', 'worker'))
+    @pytest.mark.parametrize('org, project_id', [({'id': 2, 'slug': 'org2'}, 2)])
+    def test_if_supervisor_or_worker_cannot_see_project(self, users, org, project_id, find_users, is_project_staff, role, **kwargs):
+        non_admins = find_users(role=role, exclude_privilege='admin', org=org['id'])
+        assert non_admins is not None
+        non_admin = non_admins[0]
+
+        self._test_response_403(non_admin['username'], project_id, org_id=org['id'])
+
+    # [organization] (Organization context) Member of organization that has role  maintainer or owner
+    # [organization]  can see any project even he has not any ownerships for this project
+    # or project assignee (GET /projects/{id}).
+    @pytest.mark.parametrize('role', ('maintainer', 'owner'))
+    @pytest.mark.parametrize('org, project_id', [({'id': 2, 'slug': 'org2'}, 3)])
```

Comment:
It would be great if we didn't use a hard-coded project_id, but instead looked for the right project inside the test.